bin/resty

```diff
@@ -47,6 +66,8 @@ GetOptions("c=i",             \(my $conns_num),
            "V|v",             \(my $version))
    or usage(1);
 
+check_arg_format('--shdict', qr/^[a-z]*\s\d+(?i)[km]$/, 'NAME SIZE', @shdicts);
```

Comment:
Yes, this regex was totally overlooked, my bad.

> The size units k and m are optional. default to bytes.

I was aware of this according to the nginx docs but somehow it seems not to be the case?

```
resty -e "" --shdict="dogs 1024"
nginx: [emerg] invalid lua shared dict size "1024" in /var/folders/g0/yjhyxj6d4nz4vdfs2hvwy5l80000gn/T/t1XZsHoN75/conf/nginx.conf:53
```

Also: should we allow capital letters in the shdict name as well?